spec/requests/cats_request_spec.rb

```diff
@@ -81,4 +81,42 @@
 
   end
 
+  it "doesn't create a cat without a name" do
```

Comment:
Looks like you need validations for age and enjoys too!

Reply:
added test validations for age and enjoys